STARTING.md

```diff
@@ -0,0 +1,274 @@
+<p align="center"><img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px"></p>
+<h1 align="center">OpenSearch start contributing guide</h1>
+
+This guide applies to all repositories of the OpenSearch project and should help beginners to join contributing.
+
+- [Getting started guide](#getting-started-guide)
+  - [Choose a repository](#choose-a-repository)
+  - [Install development environment](#install-development-environment)
+  - [Contribute!](#contribute)
+- [Contributing](#contributing)
+  - [What actions contribute](#what-actions-contribute)
+    - [Issues](#issues)
+    - [Pull requests](#pull-requests)
+  - [Github](#github)
+  - [Git](#git)
+- [First pull request](#first-pull-request)
+  - [Changelog](#changelog)
+
+## Getting started guide
+
+This guide is intended for all those who want to start contributing.
+
+It is distinguished by the fact that the information is summarised and will help you move forward from the start.
+
+This guide describes superficial cases to understand how contributing works
+You should definitely familiarize yourself with [contributing guide](CONTRIBUTING.md) after studying this markdown (.md).
+
+### Choose a repository
+
+Opensearch has many [repositories](https://github.com/opensearch-project).
+You can choose any of them and start contributing.
+
+Here are some options to help you decide which repository to choose:
+
+- **Choose a repository that you find interesting to explore and develop**
+- **Choose a repository associated with your technology stack**
+- **Choose a repository that has enough issues to solve**
+- **Choose a repository that does not have enough problems to solve if you like to suggest new features or find issues to solve**
+
+### Install development environment
+
+You have chosen a repository.
+
+Don't forget to make a fork of the repository you've chosen, it's written about in _DEVELOPER_GUIDE.md_
+
+The development environment (project) must now be installed to start making a contribution.
+On the repository page you can find and read [developer guide](DEVELOPER_GUIDE.md)
+
+### Contribute!
+
+Have you installed and been able to run the development environment?
+
+**Congratulations! You can start contributing!**
+
+## Contributing
+
+### What actions contribute
+
+#### Issues
+
+**Issues** is a functionality on the GitHub platform that allows users to track and manage tasks, bugs, feature requests and <br> other project-related issues. It provides a collaborative environment where developers can discuss and solve problems related to a particular repository.
+
+_Note_: Since you are a beginner, we advise you to start with simple issues.
```

Comment:
Good callout to bring attention to the good first issue label. This is definitely something we can incorporate into our current guides.